gameEngine/app.php

```diff
@@ -44,7 +44,7 @@
 if (!empty($_POST['action']) && 'update' == $_POST['action']) {
     $gameHandler->updatePing($_POST['room']);
     $gameHandler->removeInactivePlayer($_POST['room']);
-    $allMoves = $gameHandler->getMoves();
+    $allMoves = $gameHandler->getMoves(); // your session starts here
```

Comment:
I understand that you need to start session in the `getMoves()` method, accounting your game logic. But you also need to start it in the `removeInactivePlayer()` ;)
`php` complains about calling `session_start()` twice in a row (it throws a warning about that). to eliminate warning you can use the trick:

    if (PHP_SESSION_ACTIVE != session_status()) {
        session_start();
    }